### PR TITLE
Add sentry wizard setup instructions to sentry docs

### PR DESCRIPTION
**Pull request by Neo-Zhixing**
This adds a new frontmatter field, `wizard_setup`, that allows the author to specify simplified setup instructions. This will be used on the onboarding view to give simplified setup instructions.

### CODE REVIEW

**Comment by imatwawana** [comment]
Please update this page in the [contributor guide](https://docs.sentry.io/contributing/pages/frontmatter/) as well with an explanation of how to use this new frontmatter field.

**Comment by Neo-Zhixing** [comment]
@imatwawana This is not a complete feature just yet, we'll have to do an A/B test to decide if we want to keep this. And even if this passes, it's specific to onboarding wizard pages so it's not applicable to all pages.

File: package.json [hunk 1]
```diff
@@ -73,7 +73,9 @@
     "react-instantsearch-dom": "^6.4.0",
     "react-popper": "^2.2.4",
     "react-select": "^3.1.0",
+    "remark": "^10.0.1",
```
Comment (Neo-Zhixing) [inline package.json:76]: We're not exactly adding new dependencies here - these two packages are already being used by `gatsby-transformer-remark`

**Review by scefali — APPROVED** [review]

**Comment by imatwawana** [comment]
> @imatwawana This is not a complete feature just yet, we'll have to do an A/B test to decide if we want to keep this. And even if this passes, it's specific to onboarding wizard pages so it's not applicable to all pages.

Understood!

**Pull request merged**